composites/Plugin/ContentAnalysis/components/ContentAnalysis.js

```diff
@@ -219,6 +219,8 @@ class ContentAnalysis extends React.Component {
 					<AnalysisCollapsible
 						hasHeading={ true }
 						headingLevel={ headingLevel }
+						// eslint-disable-next-line no-inline-comments
+						initialIsOpen={ problemsResults.length <= 0 } /* Only open when there are no problems to be solved */
```

Comment:
Could these be less then zero?

If you create a variable above this like `problemsFound = problemResults.length` you would not need a comment either. (Also applying this for the other Results conditions below of course).

Which would make it:
```
initialIsOpen={ problemsFound == 0 }
```